src/cs/interface_visitor.ts

```diff
@@ -0,0 +1,59 @@
+import {BaseVisitor, Context} from "@apexlang/core/model";
+import {formatComment, pascalCase} from "../utils";
+import {expandType} from "./helpers";
+import {translations} from "./constant";
+
+export class InterfaceVisitor extends BaseVisitor {
+  visitInterfaceBefore(context: Context) {
+    this.write(formatComment("/// ", context.interface.description));
+    this.write(`class ${pascalCase(context.interface.name)}\n{\n`);
+    super.visitInterfaceBefore(context);
+  }
+
+  visitInterface(context: Context) {
```

Comment:
This function should emit an `interface` keyword. The `app.Map*` should move to a new visitor (e.g. `MinimalAPIVisitor`).

Reply:
ok